### PR TITLE
Initial framework commit

### PR DESCRIPTION
**Pull request by Gabib32**
Very basic structural start with with main and root files. 
Created with the cobra generator.

Signed-off-by: Gabrielle Beyer <Gabrielle.n.beyer@intel.com>

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	"github.com/spf13/cobra"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "diva",
+	Short: "DIstribution Validation Appliance",
+	Long:  `Diva is ...`,
```
Comment (matthewrsj) [inline cmd/root.go]: Might as well put *something* here.

Comment (matthewrsj) [reply cmd/root.go]: Also, call it `diva`, no need to capitalize it.

**Comment by icarus-sparry** [comment]
Shouldn't the "Signed off by" line have an Intel email address?

**Comment by Gabib32** [comment]
Working on getting a faceless account to takeout the clearlinux-pkgs account, so that my intel email can be associated with my github :)

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var cfgFile string
```
Comment (matthewrsj) [inline cmd/root.go]: Remove this, it isn't used.

Comment (Gabib32) [reply cmd/root.go]: Done

**Comment by matthewrsj** [comment]
Travis is only failing because it is configured for this repo but this PR has no .travis.yml file present. Merging.

**Pull request merged**